accordionActions.ts

```diff
@@ -0,0 +1,30 @@
+import {Action} from "@ngrx/store";
+
+export interface ChangeFormValues {
+  index:number
+  name:string
+  value:any
+}
+
+
+export enum accordionActions{

```

Comment:
for enums use `e` prefix eAccordionActions 
```suggestion
export enum eAccordionActions {

```